addons/website_event_track/static/src/scss/event_track_templates.scss

```diff
@@ -26,7 +26,6 @@
 }
 
 .o_we_online_agenda {
-    overflow-x: auto;
```

Comment:
Same error as @awa-odoo on firefox, definitely not the right fix
Actually the current state allows me to scroll.